group-coordinator/src/test/java/org/apache/kafka/coordinator/group/runtime/CoordinatorRuntimeTest.java

```diff
@@ -970,14 +970,15 @@ public void testClose() throws Exception {
         assertFalse(write2.isDone());
 
         // The coordinator timer should be empty.
-        assertEquals(0, ctx.timer.size());
+        CoordinatorRuntime.EventBasedCoordinatorTimer coordinatorTimer = ctx.timer;
```

Comment:
Was this just a cleanup? (ie, the old code would work but this is cleaner?)

Reply:
This it actually not required. Let me revert it.